Create documentation about rollback (#37)
kyma-incubator/service-catalog: docs/migration-apiserver-to-crds.md

  ./service-catalog migration --action undeploy-blocker --service-catalog-namespace=default
  ```

> ## Rollback

Reviewer: Please move this section at the end of the document.

> In case you want to revert an upgrade you may use a `helm rollback` command which will restore the Service Catalog apiserver version.

Reviewer: ```suggestion
In case you want to revert the upgrade, use the `helm rollback` command which will restore the Service Catalog API Server version.
```

> To achieve that you must delete all the Service Catalog resources and CRDs. You must also delete the unnecessary resources for the Service Catalog apiserver version. You can do it using the following commands:

Reviewer: ```suggestion
Before you proceed, you must delete all the Service Catalog resources and CRDs. You must also delete the resources that are not necessary for the Service Catalog API Server version. Use the following commands:
```

```
kubectl delete crd -l svcat=true
kubectl delete secret -n catalog catalog-catalog-webhook-cert
kubectl delete sa -n catalog service-catalog-webhook
kubectl delete sa -n catalog clean-job-account
```

> Then you can execute the rollback using the following command:

Reviewer: ```suggestion
Then you can execute the rollback using this command:
```

```
helm rollback catalog 1 --cleanup-on-fail --no-hooks
```

> After the rollback was succeeded you still have your Service Catalog resources backup from the previous upgrade stored in the persistence volume.  

Reviewer: ```suggestion
After the rollback is succeeded, you still have the backup of your Service Catalog resources from the previous upgrade stored in the persistence volume.  
```
